app/services/schools/register_ect.rb

```diff
@@ -45,7 +45,7 @@ def register!
       not_registered_as_an_ect!
 
       ActiveRecord::Base.transaction do
-        update_school_choices!
+        update_school_last_choices!
```

Comment:
`#update_last_school_choices` probably makes more sense here

Reply:
Funny. I named it initially as you suggest but them I thought it would mean: update the choices of the last school.